packs/feat-effects/effect-aegis-for-the-innocent.json

```diff
@@ -0,0 +1,43 @@
+{
+    "_id": "dFNh66uyI5yxVKbt",
+    "img": "icons/magic/light/beam-deflect-path-yellow.webp",
+    "name": "Effect: Aegis for the Innocent",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Aegis for the Innocent]</p>\n<p>You gain a +1 status bonus to AC.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 14
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder War of Immortals"
+        },
+        "rules": [
```

Comment:
Do we want to add a 10 foot aura RE?

Reply:
It doesn't say it's an aura, so I held off